projects/plugins/jetpack/class.jetpack.php

```diff
@@ -7639,4 +7639,26 @@ public static function get_products_for_purchase() {
 
 		return $products;
 	}
+
+	/**
+	 * Determine if the current user is allowed to make Jetpack purchases without
+	 * a WordPress.com account
+	 *
+	 * @return boolean True if the user can make purchases, false if not
+	 */
+	public static function current_user_can_purchase() {
+
+		// The site must be site-connected to Jetpack (no users connected).
+		if ( ! self::connection()->is_userless() ) {
```

Comment:
You'll need to update this method since we've deprecated it:
https://github.com/Automattic/jetpack/blob/master/projects/packages/connection/src/class-manager.php#L606

```suggestion
		if ( ! self::connection()->is_site_connection() ) {
```

Reply:
Thanks. Updating this too.